healthy-home/src/Views/Home/Subviews/HomeHealth/HouseGraphic/HouseGraphic.js

```diff
@@ -1,19 +1,19 @@
 import React, {useState, useEffect} from 'react';
 import axios from 'axios';
 import {connect} from 'react-redux';
-import * as actions from 'Ducks/action_creator';
+import * as actions from '../../../../../Ducks/action_creator';
 
 import './HouseGraphic.css';
 
-import niceHouse from 'Assets/HealthyHouse.png';
-import midHouse from 'Assets/MidHouse.png';
-import grossHouse from 'Assets/GrossHouse.png';
+import niceHouse from '../../../../../Assets/HealthyHouse.png';
+import midHouse from '../../../../../Assets/MidHouse.png';
```

Comment:
were using baseurl config here. you can just type Assets/healthyHouse.png etc

Reply:
if I change it to the baseurl format it breaks